lib/src/Uncertainty/Algorithm/WeightedExperiments/LHSResult.cxx

```diff
@@ -189,6 +204,8 @@ Graph LHSResult::drawCurveData(const Sample & data, const String & title) const
 
 Graph LHSResult::drawHistoryCriterion(const String & title) const
 {
+  if (!collAlgoHistory_.getSize())
+    throw InvalidArgumentException(HERE) << "No design stored!";
```

Comment:
and here: no link with the argument of the method

Reply:
The argument of the method is the graph title. We are using `CollAlgoHistory_[optimalIndex_]`  via `drawHistoryCriterion(UnsignedInteger restart, const String & title)`.